CIP-onchain-cert-metadata/README.md

```diff
@@ -0,0 +1,485 @@
+      "type": "integer",
+      "description": "Used to describe the json schema version of the off-chain metadata."
+
+    },
+    "certificationLevel": {
+      "type": "integer",
+      "description": "Describes the action this certification certificate is claiming. For the moment, CERTIFY shall be used to claim a certification that meets the Certfication Standard. AUDIT shall be used to publish on-chain an audit that may not meet the Certification standard. Shall match the on-chain certificationLevel"
+    },
+    "certificateIssuer": {
+      "type": "object",
+      "properties": {
+        "name": {
+           "type": "string",
+           "description": "Name of the Certificate Issuer."
+        },
+        "logo": {
+            "type": "string",
+            "description": "URL to the logo of the Certificate Issuer."
+        },
+        "social": {
+          "type": "object",
+          "properties": {
+            "twitter": {
+              "type": "string",
+              "description": "Twitter handle of the Certificate Issuer."
+            },
+            "github": {
+              "type": "string",
+              "description": "GitHub handle of the Certificate Issuer."
+            },
+            "contact": {
+              "type": "string",
+              "description": "Contact email of the Certification Issuer."
+            },
+            "website": {
+              "type": "string",
+              "description": "URL to the website of the Certificate Issuer."
+            },
+            "discord": {
+                "type": "string",
+                "description": "URL to the Discord server of the Certificate Issuer."
+            }
+          },
+          "description": "Social contacts of the Certificate Issuer.",
+          "required": [
+              "website",
+              "contact"
+          ]
+        }
+      },
+      "required": [
+          "name",
+          "social"
+      ]
+    },
+    "report": {
+      "type": "object",
+      "properties": {
+        "reportURLs": {
+          "type": "array",
+          "items": [
+              {
+                "type": "string",
+                "description": "A URL to the report of the certification/audit."
+              }
+          ]
+        },
+        "reportHash": {
+          "type": "string",
+          "description": "Hash of the report pointed by the links in reportURLs"
+        }
+      },
+      "required": [
+          "reportURLs",
+          "reportHash"
+      ]
+    },
+    "summary": {
+      "type": "string",
+      "description": "A string of the summary of the certification/audit. "
+    },
+    "disclaimer": {
+      "type": "string",
+      "description": "A string of the legal disclaimer from the Certificate Issuer"
+    },
+    "scripts": {
+      "type": "array",
+      "items": [
+        {
+            "smartContractInfo": {
+                "type": "object",
+                "properties": {
+                    "era": {
```

Comment:
Why is this necessary at all and if it really is, then it should be a JSON schema enum